src/assets/Azure.Core.Shared/TypeFormatters.cs

```diff
@@ -31,6 +38,12 @@ internal class TypeFormatters
             _ => throw new ArgumentException("Format is not supported", nameof(format))
         };
 
+        public static TimeSpan GetTimeSpan(JsonElement element, string format) => format switch
+        {
+            "P" => XmlConvert.ToTimeSpan(element.GetString()),
```

Comment:
Not sure there is a point in adding a format specifier if we only have a single valid format.

Reply:
Follows our current format pattern. It is basically independent of type (meaning, DateTimeOffset or TimeSpan or others). I'm sure we'll have other types that need formatting. We're at the tip of the iceberg. ⛰